### PR TITLE
Do not use time.clock(), fixes #126.

### LINKED ISSUE #126: time.clock() is deprecated
I get these warning in python 3.7

```
 /home/conda/feedstock_root/build_artifacts/fpylll_1542242253064/test_tmp/tests/test_pruner.py:33: DeprecationWarning: time.clock has been deprecated in Python 3.3 and will be removed from Python 3.8: use time.perf_counter or time.process_time instead
    tt = clock()
```

log: https://circleci.com/gh/conda-forge/fpylll-feedstock/65

### PR DESCRIPTION
**Pull request by J08nY**

### CODE REVIEW

File: tests/test_pruner.py [hunk 1]
```diff
@@ -30,22 +30,9 @@ def test_pruner():
         print(" \n GREEDY")
         radius = gaussian_heuristic(r) * 1.6
         print("pre-greedy radius %.4e" % radius)
-        tt = clock()
+        tt = process_time()
         pruning =Pruning.run(radius, overhead, r, 200, flags=Pruning.ZEALOUS, metric="solutions")
-        print("Time %.4e"%(clock() - tt))
-        print("post-greedy radius %.4e" % radius)
```
Comment (malb) [inline tests/test_pruner.py]: Why did you delete all this?

Comment (J08nY) [reply tests/test_pruner.py]: It was the same as the part before? The part for the greedy pruning test was there twice with the same constant parameters, at least it looks like that to me.

Comment (malb) [reply tests/test_pruner.py]: Ah, indeed. Thanks

**Comment by malb** [comment]
This needs a Python 2 clutch I think

**Comment by J08nY** [comment]
Yep, what do you propose? There is no adequate non-deprecated function like `time.clock` in both Python 2 and Python 3.

**Comment by malb** [comment]
It'd go for 

```python
try:
   from time import process_time # Python 3
except ImportError:
  from time import clock as process_time  # Python 2 
```

**Comment by J08nY** [comment]
> It'd go for
> 
> ```python
> try:
>    from time import process_time # Python 3
> except ImportError:
>   from time import clock as process_time  # Python 2 
> ```

This should now be fixed and ready to merge.

**Comment by malb** [comment]
Ta!
